CI build failing for OSX due to ImportError: cannot import name 'get_all_providers'

https://travis-ci.org/github/neuropoly/spinalcordtoolbox/jobs/731373490#L922

Aha. Diffing the two logs for both jobs yields a key difference:

```diff
Successfully installed Keras-2.1.5 PyQt5-sip-4.19.19 PyWavelets-1.1.1 absl-py-0.10.0 aiohttp-3.6.2 
async-timeout-3.0.1 attrs-20.2.0 bids-neuropoly-0.2 bids-validator-1.5.4 bleach-1.5.0 cachetools-4.1.1 
cffi-1.14.3 chardet-3.0.4 click-7.1.2 colored-1.4.2 coverage-5.3 cryptography-3.1.1 cycler-0.10.0 
decorator-4.4.2 dipy-1.2.0 docopt-0.6.2 future-0.18.2 futures-3.1.1 google-auth-1.22.0 
google-auth-oauthlib-0.4.1 grpcio-1.32.0 h5py-2.10.0 html5lib-0.9999999 idna-2.10 idna-ssl-1.1.0 
imageio-2.9.0 importlib-metadata-2.0.0 iniconfig-1.0.1 ivadomed-2.0.2 joblib-0.16.0 kiwisolver-1.2.0 
markdown-3.2.2 matplotlib-3.3.2 multidict-4.7.6 networkx-2.5 nibabel-3.1.1 num2words-0.5.10 oauthlib-3.1.0 
- onnxruntime-1.4.0 
+ onnxruntime-1.5.1
packaging-20.4 pandas-1.1.2 patsy-0.5.1 pillow-7.2.0 pluggy-0.13.1 protobuf-3.13.0 psutil-5.7.2 py-1.9.0 
pyOpenSSL-19.1.0 pyasn1-0.4.8 pyasn1-modules-0.2.8 pybids-0.12.1 pycparser-2.20 pyparsing-2.4.7 
pyqt5-5.11.3 pytest-6.1.0 pytest-cov-2.10.1 python-dateutil-2.8.1 pytz-2020.1 pyyaml-5.3.1 raven-6.10.0 
requests-2.24.0 requests-oauthlib-1.3.0 requirements-parser-0.2.0 rsa-4.6 scikit-image-0.17.2 scikit-learn-0.23.2
 scipy-1.5.2 seaborn-0.11.0 six-1.15.0 sqlalchemy-1.3.19 tensorboard-2.3.0 tensorboard-plugin-wit-1.7.0 
tensorflow-1.5.0 tensorflow-tensorboard-1.5.1 threadpoolctl-2.1.0 tifffile-2020.9.3 toml-0.10.1 torch-1.5.0 
torchvision-0.6.0 tqdm-4.50.0 transforms3d-0.3.1 typing-extensions-3.7.4.3 urllib3-1.25.10 werkzeug-1.0.1 
xlwt-1.3.0 yarl-1.6.0 zipp-3.2.0
```

Considering the error message is:

```
python/envs/venv_sct/lib/python3.6/site-packages/ivadomed/utils.py:8: in <module>
    import onnxruntime
python/envs/venv_sct/lib/python3.6/site-packages/onnxruntime/__init__.py:13: in <module>
    from onnxruntime.capi._pybind_state import get_all_providers, get_available_providers, get_device, set_seed, \
E   ImportError: cannot import name 'get_all_providers'
```

I'm fairly certain this version bump for `onnxruntime` is the cause.

--- a/requirements.txt
+++ b/requirements.txt
@@ -7,6 +7,7 @@ Keras==2.1.5
 matplotlib
 nibabel
 numpy
+onnxruntime==1.4.0
 pandas
 psutil
 pyqt5==5.11.3

--- a/requirements.txt
+++ b/requirements.txt
@@ -9,7 +9,6 @@ nibabel
 numpy
 # Note: If a newer version (1.5.1) is installed, Mojave CI jobs will fail
 # This is temporary until a better upstream fix can be determined. (See #2930)
-onnxruntime; sys_platform != "darwin"
 onnxruntime==1.4.0; sys_platform == "darwin"
 pandas
 psutil

--- a/requirements.txt
+++ b/requirements.txt
@@ -7,7 +7,10 @@ Keras==2.1.5
 matplotlib
 nibabel
 numpy
-onnxruntime==1.4.0
+# Note: If a newer version (1.5.1) is installed, Mojave CI jobs will fail
+# This is temporary until a better upstream fix can be determined. (See #2930)
+onnxruntime; sys_platform != "darwin"
+onnxruntime==1.4.0; sys_platform == "darwin"
 pandas
 psutil
 pyqt5==5.11.3

